front/app/containers/SearchPage/components/CrumbsBar.tsx

```diff
@@ -366,7 +366,9 @@ export default class CrumbsBar extends React.Component<
     { suggestion, suggestionValue, suggestionIndex, sectionIndex, method }
   ) => {
     const section = this.state.suggestions[sectionIndex];
-    this.props.addFilter(section.name, suggestionValue);
+    if (section.isCrowd) {
+      this.props.addFilter(section.name, suggestionValue, true);
+    } else this.props.addFilter(section.name, suggestionValue);
```

Comment:
This could be simplified to this.props.addFilter(section.name, suggestionValue, section.isCrowd) I think.